genie-web/src/main/java/com/netflix/genie/web/services/impl/LocalJobKillServiceImpl.java

```diff
@@ -48,6 +49,8 @@
 import java.io.IOException;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import java.util.Map;
+import java.util.function.Consumer;
 
 /**
  * Implementation of the JobKillService interface which attempts to kill jobs running on the local node.
```

Comment:
This comment is now out of date.

Reply:
Fixed

Reply:
@tgianos I have also moved to Runnable from Consumer<String> as the callback for kill notifications as we discussed.

```@Override
    public void registerCallbackForKill(
        @NotBlank(message = "No id entered. Unable to register job for killing.") final String id,
        @NotNull final Runnable callback) {

        jobKillCallbacks.put(id, callback);

    }```